containers/src/Kokkos_DynRankView.hpp

```diff
@@ -2010,7 +2049,8 @@ inline auto create_mirror(
   static_cast<Impl::ViewCtorProp<void, std::string>&>(prop_copy).value =
       std::string(src.label()).append("_mirror");
 
-  return dst_type(prop_copy, Impl::reconstructLayout(src.layout(), src.rank()));
+  return dst_type(prop_copy, Impl::reconstructLayout(src.layout(), src.rank()),
+                  src.impl_is_strided_from_padding());
```

Comment:
Just add `src.impl_is_strided_from_padding()` to `prop_copy` here and in the other places.

Reply:
Ok